packages/editor/src/components/post-publish-panel/index.js

```diff
@@ -83,13 +85,13 @@ export class PostPublishPanel extends Component {
 					/>
 				</div>
 				<div className="editor-post-publish-panel__content">
-					{ ! isSaving && ! isPublishedOrScheduled && (
+					{ isPrePublish && (
 						<PostPublishPanelPrepublish>
 							{ PrePublishExtension && <PrePublishExtension /> }
 						</PostPublishPanelPrepublish>
 					) }
-					{ ! isSaving && isPublishedOrScheduled && (
-						<PostPublishPanelPostpublish>
+					{ isPostPublish && (
+						<PostPublishPanelPostpublish focusOnMount={ true } >
```

Comment:
Just a side note but we should probably rename this component: it should be just `PostPublishPanel`… this is pretty confusing. 🤷‍♂️

Reply:
Yeah. Although I'm reluctant to refactor this too much given the point we are at. Perhaps this is something that can be tackled post-5.0 release when we have more leeway to modify the external API.

Reply:
Cool, works for me.